addons/mrp/i18n/mrp.pot

```diff
@@ -3816,6 +3816,8 @@ msgstr ""
 
 #. module: mrp
 #: model:ir.model.constraint,message:mrp.constraint_mrp_production_qty_positive
+#: code:addons/mrp/models/mrp_bom.py:0
```

Comment:
@mart-e is it ok for you doing like this ? thanks

Reply:
yes, will reuse existing translation for code too this way :+1: 
(a user will still need to manually update the translations to benefit from it but at least nothing needed from the translators)